web/src/beta/features/WorkspaceSetting/hooks.ts

```diff
@@ -131,13 +138,55 @@ export default () => {
     [useRemoveMemberFromWorkspace]
   );
 
+  // update a member of workspace
+  const handleUpdateMemberOfWorkspace = useCallback(
+    async ({ teamId, userId, role }: WorkspacePayload) => {
+      try {
+        if (userId && role) {
+          const { status } = await useUpdateMemberOfWorkspace(
+            teamId,
+            userId,
+            role
+          );
+          if (status === "success") {
+            console.log("Member update successfully");
+          }
+        }
+      } catch (error) {
+        console.error("Failed to update member from workspace:", error);
+      }
```

Comment:
Same as above, i think api hooks already handled the errors and notifications, we don't need to to these again where we use the api hooks.